src/components/new_id/NewID.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useMemo, Suspense, lazy } from "react";
+import { Container } from "react-bootstrap";
+import { useParams } from "react-router-dom";
+import { useTypedSelector } from "../../hooks/useTypedSelector";
+import { useActions } from '../../hooks/useActions';
+import { getDate } from "../../utils/utils";
+import {CommentsList} from './CommentsList';
+import {NewsItemType, StateType} from '../../types/types';
+import {ErrorBoundary} from 'react-error-boundary'
+import {useSelector} from "react-redux";
+
+const NewID = () => {
+    const newsItems = useTypedSelector(state => state.news);
+    const {getComments} = useActions();
+    const comments = useSelector((state: StateType) => state.comments);
+    const {id} = useParams();
+
+    useEffect(() =>{
+        getComments(Number(id));
+        const instervalId = setInterval(() =>{
+           getComments(Number(id)); 
+        }, 60000)
+        return () => clearInterval(instervalId);
+    }, [])
+
+    const newsItem = useMemo(
+        () => newsItems.news.find((item: NewsItemType) => item.id === Number(id)),
```

Comment:
приведения к числу через класс `Number` допустимо, но если что советую более легкий вариант 
Читать легче
```suggestion
        () => newsItems.news.find((item: NewsItemType) => item.id === +id),
```